wdi_radio_lab_angular/js/app.js

```diff
@@ -0,0 +1,39 @@
+
+(function(){
+angular
+  .module("WdiRadio", ["ui.router", "ngResource"])
+  .config(["$stateProvider", RouterFunction])
+  .controller("WdiRadioIndexController", [ "WdiRadioFactory", WdiRadioIndexControllerFunction])
+  .controller("WdiRadioShowController", ["WdiRadioFactory", "$stateParams", WdiRadioShowControllerFunction])
+  .factory("WdiRadioFactory", ["$resource", WdiRadioFactoryFunction]);
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("WdiRadioIndex", {
+      url: "/",
+      templateUrl: "js/ng-views/index.html",
+      controller: "WdiRadioIndexController",
+      controllerAs: "vm"
+    })
+    .state("WdiRadioShow", {
+      url: "/songs",
+      templateUrl: "js/ng-views/show.html",
+      controller: "WdiRadioIndexController",
+      controllerAs: "vm"
+    });
+  }
+
+  function WdiRadioIndexControllerFunction(WdiRadioFactory){
+    console.log("hello")
+    this.songs = WdiRadioFactory.query();
+  }
+
+  function WdiRadioShowControllerFunction(WdiRadioFactory, $stateParams){
+    this.song = WdiRadioFactory.get({id: $stateParams.id});
```

Comment:
Doesn't look like you set up a `show` action for your Rails songs controller. Need that in order for `.get` to work -- assumes you're API is restful.